src/Pages/Courses/Courses.tsx

```diff
@@ -16,7 +16,7 @@ export const Courses = (): ReactElement => {
     <div>
       {courses &&
         courses.map((el) => (
-          <Link to={""}>
+          <Link to={"/test"}>
```

Comment:
Why test? It's not a test, it's a course i think. And you redirect always to the same endpoint. Should be /course/id or /courses/id.